### ISSUE TITLE
[Listbox] Single-select receive focus

### ISSUE THREAD

**Issue by gfrancischelli** [samaraoliveiram/mula#2]
Focus should be managed like [Managing Focus in Composites Using aria-activedescendant](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_focus_activedescendant), since [roving tabindex](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex), wouldn't work when using a `combobox` (which requires focus to be on the text input element)
- [x]  If none of the options are selected before the listbox receives focus, the first option receives focus. 
  - [x] ~Optionally, the first option may be automatically selected~ _we decided to not implement this on the mvp_
- [x]  If an option is selected before the listbox receives focus, focus is set on the selected option.
- [x] If an option is selected, the selected option receives focus
